src/commands/hld/README.md

```diff
@@ -0,0 +1,84 @@
+# HLD
+
+Initialize a Bedrock HLD(High Level Definition) repository and deploy pipelines
+to materalize manifests.
+
+Usage:
+
+```
+spk hld [command] [options]
+```
+
+Commands:
+
+- [HLD](#hld)
+  - [Requirements](#requirements)
+  - [Commands](#commands)
+    - [init](#init)
+    - [install-manifest-pipeline](#install-manifest-pipeline)
+
+Global options:
+
+```
+  -v, --verbose        Enable verbose logging
+  -h, --help           Usage information
+```
+
+## Requirements
+
+There are a few base assumptions that `spk` makes, as this will affect the set
+up of pipelines:
+
+1. Both HLD and manifest repositories are within a single Azure DevOps project.
+1. The access token being utilized via `spk` has access to both repositories.
```

Comment:
Not sure if we tell folks how to generate tokens. Maybe a link to https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops